recipients_app/android/app/src/profile/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="org.socialincome.app">
```

Comment:
package='org.socialincome.app' is replace by the namespace 'org.socialincome.app' in build.gradle